### ISSUE TITLE
Launching Jitsi in the small window by default sometimes has poor UX due to Chrome dialogues

### ISSUE THREAD

**Issue by DHaussermann** [mattermost/mattermost-plugin-jitsi#122]
When you enable `Embed Jitsi video inside Mattermost` it seen to default to the smaller corner window rather than the full window. This makes for a cleaner UX but, there are some cases where it's not very usable because Chrome opens a dialogue to prompt you for access to camera and mic or to inform you of a jitsi plugin you can install.

It is then not possible to use the window without dismissing the dialogue and it is hard to dismiss it without full screening the window.

Not sure how much can be done about this or if there is a way to improve this UX. Note that the user can click to not that this may only occur on the 1st meeting launch. Once the user has granted access for Jisti to use the mic and camera via Chrome it should not ask again. The user can also click a box to no longer get prompted about the Jitsi plugin. 

Steps:
- Setup Jisti
- Enable `Embed Jitsi video inside Mattermost`
- Launch desktop in a fresh install state (having never used the Jitsi app) 
- Connect to server from desktop
- Join a meeting
**Observed** Small corner window opens and Chrome dialogue takes up all the space.
![Screen Shot 2020-06-24 at 2 21 22 PM](https://user-images.githubusercontent.com/35588783/85623159-f9482c00-b635-11ea-802a-2cdab8ba7366.png)

**Comment by jespino** [comment]
yes, probably we can handle that when we get in the `minimized-mode` in the Jitsi upstream.

**Comment by jespino** [comment]
As far as this is part of the embedded (experimental) feature, and it depends a lot on the Jitsi side of the things, I going to defer it until 2.1.0 (Unless @jasonblais is against it?)

**Comment by jasonblais** [comment]
I'm okay with v2.1, but this is a great find Dylan for us to improve on
